source/fundamentals/data-formats/document-data-format-extended-json.txt

```diff
@@ -119,146 +119,281 @@ corresponds to the format of the example you want to see:
+      ``BsonDocument`` by calling the ``toJson()`` method. You can pass it an
+      instance of ``JsonWriterSettings`` to specify the Extended JSON format.
+      
+      In this example, we output the Extended JSON in the "Relaxed" format.
+      
+      .. code-block:: java
+      
+         Document myDoc = new Document();
+         myDoc.append("_id", new ObjectId("507f1f77bcf86cd799439012")).append("myNumber", 11223344);
+      
+         JsonWriterSettings settings = JsonWriterSettings.builder().outputMode(JsonMode.RELAXED).build();
+         System.out.println(doc.toJson(settings));
+      
+      The output of this code example resembles the following:
+      
+      .. code-block:: none
+         :copyable: false
+      
+         {"_id": {"$oid": "507f1f77bcf86cd799439012"}, "myNumber": 11223344}
+
+   .. tab:: BSON Library
+      :tabid: bson-library-write
+
+      You can also output an Extended JSON string from data in Java objects using
+      the BSON library with the :java-docs:`JsonWriter </apidocs/bson/org/bson/json/JsonWriter.html>`
+      class. To construct an instance of ``JsonWriter``, pass a subclass of a Java
+      ``Writer`` to specify how you want to output the Extended JSON. You can
+      optionally pass a :java-docs:`JsonWriterSettings </apidocs/bson/org/bson/json/JsonWriterSettings.html>`
+      instance to specify options such as the Extended JSON format. By default, the
+      ``JsonWriter`` uses the "Relaxed" Extended JSON format. MongoDB driver's
+      document classes also use this class to convert BSON to Extended JSON.
+      
+      The following code example shows how you can use ``JsonWriter`` to create an
+      Extended JSON string and output it to ``System.out``. We specify the format
+      by passing the :java-docs:`outputMode() </apidocs/bson/org/bson/json/JsonWriterSettings.Builder.html#outputMode(org.bson.json.JsonMode)>`
+      builder method the ``JsonMode.EXTENDED`` constant:
+      
+      .. code-block:: java
+      
+         JsonWriterSettings settings = JsonWriterSettings.builder().outputMode(JsonMode.EXTENDED).build();
+      
+         try (JsonWriter jsonWriter = new JsonWriter(new BufferedWriter(new OutputStreamWriter(System.out)), settings)) {
+             jsonWriter.writeStartDocument();
+             jsonWriter.writeObjectId("_id", new ObjectId("507f1f77bcf86cd799439012"));
+             jsonWriter.writeInt64("myNumber", 11223344);
+             jsonWriter.writeEndDocument();
+             jsonWriter.flush();
+         }
+      
+      The output of this code example resembles the following:
+      
+      .. code-block:: none
+         :copyable: false
+      
+         {"_id": {"$oid": "507f1f77bcf86cd799439012"}, "myNumber": {"$numberLong": "11223344"}}
+
+   .. tab:: JsonObject Class
+      :tabid: jsonobject-class-write
+
+      The ``JsonObject`` class is a wrapper for JSON strings. You can access the
```

Comment:
I think you've already mentioned the same thing in the prior section on reading from JsonObject. You might consider replacing with a more specific description of what it enables you to do in the context of writing data.

Reply:
Thanks for feedback. This was a consequence of incorrect placement, and was resolved by moving to documents section.